src/components/Crews/findMultipleRounds.js

```diff
@@ -15,3 +15,7 @@ export default teams =>
         }, []),
         'distance'
     );
+    // If only 1 round in regatta
+    if (RoundsList.length === 1) return [];
```

Comment:
I do notice that having brackets tends to be more consistent and easier in the future.